src/components/SearchList/search-list.css

```diff
@@ -0,0 +1,25 @@
+.search-list {
+  position: absolute;
+  z-index: 2;
+  background-color: var(--white);
+  width: 100%;
+  height: fit-content;
+  max-height: 50vh;
+  overflow-y: scroll;
+  box-shadow: 1px 2px 3px rgba(43, 43, 43, 0.603);
+}
+
+.search-card-image {
+  width: 15%;
+  flex-shrink: 0;
+}
+
+.search-card-image img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.search-card-title {
```

Comment:
If this class is empty, Better to remove it

Reply:
Changes made.